Fix issue number 13 and 18

mrabdulrrahimklis opened the pull request:
encryption on file download added
decryption is on hold TODO note added on file upload

mrabdulrrahimklis committed:
Fix issue number 13 and 18

encryption on file download added
decryption is on hold TODO note added on file upload

TimvanSteenbergen reviewed (commented):
SHA512 stands for SecureHashingAlgorithm. It is only hashing, so not suitable for encryption/decryption. AES is probably better, I found just now: http://www.movable-type.co.uk/scripts/aes.html but I will ask some expert on cryptography. 
Let's forget this issue 18 and move on to issues 54 and 48:
•	Issue 54: make Opfes multiple-language. Just setting up the translation tool, the translation-file containing three or four textsvariables and three or four usages of those textvariables in the code. I will search the code for all needed replacements myself.
•	Issue 48: Passwordfields that are hidden don't trigger Opfes.

I have one branch for 54 and another branch for 48, so you won't have any merge problems.

TimvanSteenbergen committed:
investigation of AES javascript options

TimvanSteenbergen committed:
Added encrypting and decrypting. It encrypts, but still struggling with:
- key-types mismatching and
- how to call decrypt and know what iv to use? Or what counter to use?

TimvanSteenbergen commented:
![image](https://user-images.githubusercontent.com/3594338/27849117-cbdd62fa-614a-11e7-81a0-47de1a058031.png)

TimvanSteenbergen reviewed (commented):
Ok, I pulled the branch, I ran 'npm update' and regenerated the typescript-files. Result the builds-directory is there. Then I have reloaded the extension in the browser. Now the button ChangeToFrance does not trigger any functionality, the button has no click-event assigned. The other buttons do.
